### DISCUSSION TITLE
Welcome to FDPClient Discussions!

### DISCUSSION THREAD

**Discussion by contionability** [SkidderMC/FDPClient#680] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JaveMave** [comment]
Is it possible to change the autodisable mode (for example from "FLAG" to "NONE") in a script? For the State i just use Fly.setState(true). But something like this doesnt work for autodisable.
I am really stupid btw.

**Reply by C00LC0D35** [reply]
you can do Fly.setState(false) on a s08 packet (flag) or you can send command ".ad fly flag"

**Reply by Fdp5** [reply]
5.3.5.jar vernite верните пожалуйста ото моя самая любимая версия 1.8.9  нечего неизменяйте просто верните 5.3.5.jar

**Reply by Fdp5** [reply]
Сосной можно свезаца через дс artem_ruzanov1234_51974

**Comment by algrowww** [comment]
Hello sir, I hope you're doing well. I appreciate your work. However, I have encountered a problem. My friend from SMP (Survival Multiplayer) suggested that I share my screen with them, but since I'm using an FDP client, I am unable to do so. Therefore, I need your help with three things. Firstly, could you please guide me on how to change the icon? Secondly, I would like to know how I can change my name. Whenever I open the game, it displays my FDP name and a GitHub link. Lastly, I'm wondering how I can change the background to a normal one. I'm eagerly awaiting your response and I would be extremely grateful for your assistance. Thank you for your kindness in advance.
